ship-it-inc/ship-it-fend
# #168090270 Implement login functionalities

#### What does this PR do?
It implements login functionalities
#### Description of Task to be completed?
- add ability to login
- add error toaster
#### How should this be manually tested?
- Fetch the branch `ft-login-functionalities-168090270`
- Check out into the branch `ft-login-functionalities-168090270`
- run npm run dev
- You should be to login with email.
#### Any background context you want to provide?
Nil
#### What are the relevant pivotal tracker stories?
[#168090270](https://www.pivotaltracker.com/story/show/168090270)
#### Screenshots (if appropriate)
Nil
#### Questions:
Nil
